packages/components/src/components/radio/radio.stories.ts

```diff
@@ -1,14 +1,20 @@
 import '../../solid-components';
+import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 
 const { argTypes, parameters } = storybookDefaults('sd-radio');
 const { generateTemplate } = storybookTemplate('sd-radio');
 const { overrideArgs } = storybookHelpers('sd-radio');
 
+/**
+ * Allows the user to select a single option from a group.
+ */
+
 export default {
   title: 'Components/sd-radio',
+  tags: ['!dev'],
   component: 'sd-radio',
-  args: overrideArgs([{ type: 'slot', name: 'default', value: 'Default Slot' }]),
+  args: overrideArgs([{ type: 'slot', name: 'default', value: 'Radio' }]),
```

Comment:
I still see as Text "Default Slot" but I would like to see "Radio" next to the circle ... 
<img width="484" alt="Bildschirmfoto 2024-08-10 um 21 01 26" src="https://github.com/user-attachments/assets/fdd264ec-acb4-4322-aa41-9327d7c9d794">